### PR TITLE
(chore): Add packaging for Meteor.js

### PR DESCRIPTION
**Pull request by Urigo**
Hi

I'm a package maintainer for [Meteor.js](https://www.meteor.com/), a popular [full-stack JavaScript framework](https://github.com/meteor/meteor).

I'm also the creator of [angular-meteor](http://angularjs.meteor.com/), a library that let's you work with AngularJS on top of Meteor.

This PR will allow you to directly publish updated versions of angulartics as they become available.
I've already published the current version of the package on Atmosphere (Meteor's package directory). 

To make this under your username all you have to do is create an account at https://meteor.com/ (click SIGN IN, then Create account). After you've done that, you will be able to publish the package under you name and I will delete mine.

There are also grunt scripts to help you publish to Meteor automaticly, here is an example:
https://github.com/MeteorPackaging/ui-router/commit/8f2e39e9780dee19201b4982ff6bbd98cf9f8770

Thanks & best regards,
Uri

### CODE REVIEW

**Comment by lunks** [comment]
Do I have to manually publish to something like RubyGems or does it fetch versions from git automagically?

I'm okay adding this to the repository as long as we don't need to publish a new version ourselves -- the owner is not active on the repo anymore, and I'd rather not have the chance of forgetting uploading a new version to meteor.

**Comment by Urigo** [comment]
Hi!  thanks for responding.
You can add a grunt task for publishing new versions like in the ui-router's example

**Comment by luisfarzati** [comment]
@Urigo Sorry guys, but we'll pass adding support for all the package managers out there until we see some popular interest from the community. npm and bower (but specially npm) support should be enough for now. Please see #377 and check out https://github.com/angulartics/angulartics-google-analytics as we are moving all plugins out of the core and **packaging for npm in the process**. 

@markvp, we are dropping NuGet as well.

Of course feel free to manage this on your side if you are willing to.

File: package.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  name: packageName,
+  version: version,
+  summary: 'angulartics (official): Analytics for AngularJS applications',
+  git: 'git@github.com:luisfarzati/angulartics.git'
+});
+
+Package.onUse(function(api) {
+  api.versionsFrom(['METEOR@0.9.0', 'METEOR@1.0']);
+
+  api.use('angular:angular@1.1.5', where);
```
Comment (luisfarzati) [inline package.js]: Once we package for npm both Angulartics core and all its plugins, shouldn't be as easy as doing what you are doing here for requiring Angular?

Comment (Urigo) [inline package.js]: No because the `angular:angular` is something that we are releasing right now

**Pull request closed without merging**